site/docs/v1/tech/oauth/_access-token-claims.adoc

```diff
@@ -0,0 +1,57 @@
+[.api]
+[field]#applicationId# [type]#[UUID]#::
+The unique Id of the Application for which the User has been authenticated. A JWT can only represent authorization to a single Application.
++
+This claim is only present if the User has a registration to the Application.
++
+To obtain a JWT for another Application you must either authenticate again with a different `applicationId` using the link:/docs/v1/tech/apis/login#authenticate-a-user[Authentication] API or utilize the link:/docs/v1/tech/apis/jwt#issue-a-jwt[Issue a JWT] API to exchange a valid JWT for another.
+
+[field]#aud# [type]#[String]#::
+The audience the JWT is intended for. This registered claim is defined by https://tools.ietf.org/html/rfc7519#section-4.1.3[RFC 7519 Section 4.1.3].
++
+This claim will be equal to the `client_id`.
+
+[field]#authenticationType# [type]#[String]#::
+The method used to authenticate the User which resulted in this JWT being generated. The possible values are:
++
+include::_authentication_type_claim_values.adoc[]
+
+[field]#auth_time# [type]#[Long]# [since]#Available since 1.36.0#::
+The time of the initial authentication request, expressed as UNIX time which is the number of seconds since Epoch. This claim will remain the same even when the token has been re-issued through the use of a Refresh Token.
+
+[field]#email# [type]#[String]#::
+The email address of the User whose claims are represented by this JWT.
+
+[field]#email_verified# [type]#[Boolean]#::
+The OpenId Connect claim indicating if the User's email has been verified.
+
+[field]#exp# [type]#[Long]#::
+The expiration instant of the JWT, expressed as UNIX time which is the number of seconds since Epoch. This registered claim is defined by
+https://tools.ietf.org/html/rfc7519#section-4.1.4[RFC 7519 Section 4.1.4].
+
+[field]#iat# [type]#[Long]#::
+The instant that the JWT was issued, expressed as UNIX time which is the number of seconds since Epoch. This registered claim is defined by
+https://tools.ietf.org/html/rfc7519#section-4.1.6[RFC 7519 Section 4.1.6].
+
+[field]#jti# [type]#[String]# [since]#Available since 1.18.0#::
+The unique identifier for this JWT. This registered claim is defined by
+https://tools.ietf.org/html/rfc7519#section-4.1.7[RFC 7519 Section 4.1.7].
+
+[field]#iss# [type]#[String]#::
+The issuer of the JWT. For FusionAuth, this is always the value defined in the tenant JWT configuration. This registered claim is defined by
+https://tools.ietf.org/html/rfc7519#section-4.1.1[RFC 7519 Section 4.1.1].
+
+[field]#preferred_username# [type]#[String]# [since]#Available since 1.5.0#::
+The username of the User whose claims are represented by this JWT.
```

Comment:
should user be caps?

Reply:
We do caps sometimes when it is a FusionAuth concept, rather than a normal entity. We should be better about that.